projects/cm_mcu/InitTask.c

```diff
@@ -31,14 +30,19 @@ void InitTask(void *parameters)
   ROM_SysCtlResetCauseClear(r);
   errbuffer_put(EBUF_RESTART, restart_reason);
   log_info(LOG_SERVICE, "REC register=0x%08x\r\n", restart_reason);
+  init_registers_ff(); // initalize I/O expander for fireflies -- with FF monitoring via I2C in other threads, it grabs semaphore inside
 
-  init_registers_clk(); // initalize I/O expander for clocks
+  init_registers_clk(); // initalize I/O expander for clocks -- with clock monitoring via I2C in other threads, it grabs semaphore inside
   log_info(LOG_SERVICE, "Clock I/O expander initialized\r\n");
 #ifdef REV2
-  init_load_clk(0);
-  init_load_clk(2);
+  init_load_clk(0); // load clock r0a config from EEPROM
```

Comment:
replace with loop?

Reply:
clock r0b and r1c do not have .txt or .csv config files for me to load from so I just kept them the way they are. I will ask Charlie to generate them and change this to for loop.